pallets/oracle/res/contract.abi

```diff
@@ -0,0 +1,33 @@
+[{
+  "constant": true,
```

Comment:
This file .abi is the ABI of the chainlink smart contract ([here](https://etherscan.io/address/0x1c07afb8e2b827c5a4739c6d59ae3a5035f28734#code)). I intend to use it in the offchain worker with ethabi to create its encoded version and use it to call `eth_call` (at the moment is done manually - L275 in oracle/src/lib.rs)